src/main/java/eggventory/StockList.java

```diff
@@ -32,6 +32,33 @@ public StockList() {
         return stockList;
     }
 
+    /**
+     *
+     * @param name
+     */
+    public void addStockType(String name) {
+        stockList.add(new StockType(name, false));
```

Comment:
eh we'll have to remove the unique/collective stuff from stocktype and move to stock